chart/openfaas/values.yaml

```diff
@@ -146,24 +146,30 @@ gateway:
     successThreshold: 1
 
 faasnetesPro:
-  image: ghcr.io/openfaasltd/faas-netes:0.2.9
+  image: ghcr.io/openfaasltd/faas-netes:0.3.0
 
 operatorPro:
-  image: ghcr.io/openfaasltd/faas-netes:0.2.9
+  image: ghcr.io/openfaasltd/faas-netes:0.3.0
 
-faasnetes:
-  image: ghcr.io/openfaas/faas-netes:0.16.7
+
+functions:
   imagePullPolicy: "Always"    # Image pull policy for deployed functions
   httpProbe: true              # Setting to true will use HTTP for readiness and liveness probe on function pods
   setNonRootUser: false        # It's recommended to set this to "true", but test your images before committing to it
   readinessProbe:
     initialDelaySeconds: 2
```

Comment:
Apparently initialDelaySeconds can be 0 now, so we should do that in the chart.

Reply:
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes